abc.js

```diff
@@ -0,0 +1,368 @@
+    p4: 'This hero has high potential of magical damage he can nuke initiate can escape certain has many skills with low cooldown',
+    p5: 'Ember spirit can play both as a magical or physical hitter he can can play roles to initiate destroy back end supports has high physical damage',
+    p6: 'Morphling is a carry hero with ability to change his stats make him hard to kill can nuke enemy heros has burst of high magical and physical damage and can escape certain deaths',
+    p7: 'Crystal Maiden is a support hero who has high early magical damage is a very good early lane support',
+    p8: 'Lone druid is a very versatile hero. His bear and himself play together and because of low attack time it makes him very scary carry against a specific pick',
+    p9: 'juggernaut is one of the most reliable carry of dota2 who has always been a priority for the players can take away the games from the opponent mouth at times.',
+
+  },
+  {
+    h2Class: 'top-heading',
+    topDivClass: 'top-div',
+    borderClass: 'speaker-border',
+    imgClass: 'img1',
+    imgcontainer: 'img-container',
+    paraContainer: 'para-container',
+    imgHeading1: 'img-heading',
+    imgPara: 'img-para',
+    img2: 'img2',
+    imgContainer2: 'img-imgContainer2',
+    imgHeading2: 'img-heading2',
+    img2paraDiv: 'img2-para-div',
+    img2para2: 'img2-para2',
+    more: 'more',
+    morePara: 'more-para',
+    span: 'material-symbols-outlined',
+    desktopImgConatainer: 'desktop-img-container',
+    moreHerosContainer: 'more-hero-container',
+    secondDivisionOfHeros: 'second-division-of-heros',
+    invoker: 'invoker',
+    ember: 'ember',
+    morph: 'morph',
+    crystalMaiden: 'crystal-maiden',
+    loneDruid: 'lone-druid',
+    juggernaut: 'juggernaut',
+    hero3heading: 'hero3',
+    hero4heading: 'hero4',
+    hero5heading: 'hero5',
+    hero6heading: 'hero6',
+    hero7heading: 'hero7',
+    hero8heading: 'hero8',
+    hero3Img: 'hero3-img',
+    hero4Img: 'hero4-img',
+    hero5Img: 'hero5-img',
+    hero6Img: 'hero6-img',
+    hero7Img: 'hero7-img',
+    hero8Img: 'hero8-img',
+    hero3Attributes: 'hero3-Attributes',
+    heroDetails: 'hero-Details',
+    thirdDivisionOfHeros: 'third-Division-Of-Heros',
+    fourDivisionOfHeros: 'fourth-Division-Of-Heros',
+    headingImgParaDiv: 'heading-Img-Para-Div5',
+
+  },
+  {
+    heroAttribute1: 'Intelligence, Core, Carry, Nuker',
+    heroAttribute2: 'Agility, Core, Carry, Escaper',
+    heroAttribute3: 'Intelligence, Core, Carry, Escaper, Nuker, Disabler',
+    heroAttribute4: 'Agility, Core, Carry, Escaper, Disabler',
+    heroAttribute5: 'Agility, Core, Carry, Escaper,Nuker',
+    heroAttribute6: 'Intelligence, Support',
+    heroAttribute7: 'Agility, Core, Carry, Escaper, Durable',
+    heroAttribute8: 'Agility, Core, Carry, Escaper',
+  },
+  {
+    hero3: 'invoker',
+    hero4: 'Ember',
+    hero5: 'Morph',
+    hero6: 'Crystal Maiden',
+    hero7: 'Lone Druid',
+    hero8: 'Juggernaut',
+  },
+
+];
```

Comment:
- [ ] As mentioned by the previous reviewer, your featured speakers array list should contain objects of the featured speakers and this objects should be looped to create the featured section dynamically. The featured array should be something like;
```jabascript
const speakers = [
  {
    name: "speaker1",
    image: "link/to/image.png",
    role: "Technician",
    Description: "Big the guy working in big tech company",
  },
  {
    name: "speaker2",
    image: "link/to/image.png",
    role: "Technician",
    Description: "Big the guy working in big tech company",
  },
  {
    name: "speaker3",
    image: "link/to/image.png",
    role: "Technician",
    Description: "Big the guy working in big tech company",
  },
];
```

Then you loop this to create your markup on load of the document like;
```javascript
window.onload = () => {
  speaker.forEach((speaker) => {
    .... code to build HTML for one speaker goes here
  });
}
```

#### Note
Your code must not be exactly like this, this is just to give you a general idea of what is required of you.

Reply:
alright i have done so much hard work on this and as now I set previous code to display none and have created a new js file because I had two options now either to create js file from scratch or just create a new one so I added a new one added used  for loop as was asked by the previous reviewer to iterate objects using loop. so hope I m done. Thanks for your kind feedback